openedx/core/djangoapps/credentials/utils.py

```diff
@@ -15,13 +18,27 @@ def get_credentials_records_url(program_uuid=None):
     Arguments:
         program_uuid (str): Optional program uuid to link for a program records URL
     """
-    base_url = CredentialsApiConfig.current().public_records_url
```

Comment:
This function was using a deprecated settings field (`public_records_url` from the CredentialsApiConfig).

I have updated the function to use the recommended non-deprecated setting (`CREDENTIALS_PUBLIC_SERVICE_URL`).